.docker/dev.Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM registry.tudelft.nl/samxl/scancontrol:humble-base AS dev 
+ARG TARGET_WS
```

Comment:
Why is this needed as an input? For the user of this service the location of the workspace inside the container does not matter.

Suggestion: set a default value for the `TARGET_WS`